### PR TITLE
Add buttons to view notebook sources and run in colab

### PR DESCRIPTION
**Pull request by wojciechowskidawid**
This PR extends notebooks with a buttons to view sources and run in colab.

### CODE REVIEW

File: examples/hello-world/hello_world.py [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # %% [markdown]
 """
 ![Renode](https://antmicro.com/OpenSource/assets/images/projects/renode.png)
+<table class="tfo-notebook-buttons" align="left">
```
Comment (PiotrZierhoffer) [inline examples/hello-world/hello_world.py:4]: What is this class?

File: examples/hello-world/hello_world.py [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+  <td>
+    <a target="_blank" href="![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/antmicro/tensorflow-zephyr-vexriscv-examples/blob/main/examples/hello-world/$NOTEBOOK.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>
```
Comment (PiotrZierhoffer) [inline examples/hello-world/hello_world.py:6]: This doesn't look like a correct href. Does it work?

File: examples/magic-wand/magic_wand.py [hunk 3]
```diff
@@ -1,6 +1,17 @@
 # %% [markdown]
 """
 ![Renode](https://antmicro.com/OpenSource/assets/images/projects/renode.png)
+<table class="tfo-notebook-buttons" align="left">
```
Comment (PiotrZierhoffer) [inline examples/magic-wand/magic_wand.py:4]: Same remarks as above

**Review by PiotrZierhoffer — CHANGES_REQUESTED** [review]

File: examples/hello-world/hello_world.py (hunk 1 above)
Comment (wojciechowskidawid) [reply examples/hello-world/hello_world.py:4]: It comes from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/examples/micro_speech/train/train_micro_speech_model.ipynb?short_path=1a1d5ff#L41, and I can see this is not the only one case with that class.
Anyway, You're right and there is no any css with that class, and using it do literally nothing, so I think I would drop this one.

**Pull request merged**